src/otx/algorithms/detection/adapters/openvino/task.py

```diff
@@ -446,14 +446,17 @@ def load_inferencer(
             ):
                 args.append({"resize_type": "fit_to_window_letterbox", "pad_value": 114})
             inferencer: BaseInferencerWithConverter = OpenVINODetectionInferencer(*args)
-        if self.task_type == TaskType.INSTANCE_SEGMENTATION:
-            if self.config.tiling_parameters.enable_tiling:
+        if self.task_type == TaskType.INSTANCE_SEGMENTATION or self.task_type == TaskType.ROTATED_DETECTION:
+            if not self.config.tiling_parameters.enable_tiling:
```

Comment:
Some of models have keep_ratio w/ True. Is it ok to use standard for all i-seg models?

Reply:
Thanks for noticing. I missed that, for non-tiling pipelines keep_ratio is False now